Sources/Alchemy/SQL/QueryBuilder/Query.swift

```diff
@@ -43,18 +50,39 @@ public class Query: Sequelizable {
         return self
     }
 
-    public func from(table: String, as alias: String? = nil) -> Self {
-
-        //TODO: Allow for selecting from subqueries
-
-        guard let alias = alias else {
-            self.from = table
-            return self
-        }
-        self.from = "\(table) as \(alias)"
+    /// Set the table to perform a query from.
+    ///
+    /// - Parameters:
+    ///   - table: The table to run the query on
+    /// - Returns: The current query builder `Query` to chain future queries to
+    public func table(_ table: String) -> Self {
+        self.from = table
         return self
     }
 
+    /// An alias for `table(_ table: String)` to be used when running
+    /// a `select` query that also lets you alias the table name.
+    ///
+    /// - Parameters:
+    ///   - table: The table to select data from
+    ///   - alias: An optional alias to use in place of table name
```

Comment:
I've been adding a "Defaults to `___`." wherever there are defaults. Some other spots in this file too